### PR TITLE
Use case view Requests and case

### PR DESCRIPTION
**Pull request by ISMAELCHONA04**
Finalized use case for viewing requests given a user. The Test passes if the file requestTest.csv is empty excluding the 
header.

### CODE REVIEW

**Review by michellengnx — CHANGES_REQUESTED** [review]
Please exclude` .idea/* `from your commit

File: Dockerfile
Comment (michellengnx) [inline Dockerfile:1]: No Dockerfile change

File: csc207-project.iml
Comment (michellengnx) [inline csc207-project.iml:1]: No config file change

File: src/pom.xml
Comment (michellengnx) [inline src/pom.xml:1]: Do not change my pom.xml file 😠

**Review by michellengnx — CHANGES_REQUESTED** [review]

**Review by izorakayumova — CHANGES_REQUESTED** [review]
Very informative commits and your packaging structure looks great except for your one test - can you refactor it to test/java/data_access. Please include javadoc for everything that is publicly documented.

**Comment by michellengnx** [comment]
I am not merging this pull request until all comments are resolved.
- No config files should be committed.
- Do not touch Docker and pom.xml files
- Do NOT use absolute path in your code.
- Please check your log and ensure that you code at least run.

If you have any questions regarding how Maven project is structured, please see [here](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html).

**Comment by michellengnx** [comment]
Code not fixed, closing this pull request. Once you fix your code and get it up to date with develop branch, feel free to reopen it.

**Pull request closed without merging**